sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -157,6 +207,15 @@ def plot(self, ax=None, *, name=None, **kwargs):
         ylabel = "Precision" + info_pos_label
         self.ax_.set(xlabel=xlabel, ylabel=ylabel)
 
+        if plot_chance_level:
```

Comment:
We only need to check that the prevalence is defined here and we don't need for a comment because the code is self explanatory.

```suggestion
        if plot_chance_level and self.prevalence is not None:
```